python/test/t_FORM_interval.py

```diff
@@ -62,6 +62,8 @@
     ot.RandomGenerator.SetSeed(0)
     # algo = getattr(openturns, algoName)(event)
     algo = ot.FORM(ot.Cobyla(), event, X.getMean())
+    algo.setLimitStateTolerance(2.)
```

Comment:
??? Strange value for a tolerance. Maybe the case where the event is always true should be treated separately?
BTW prefer 2.0 to 2., it makes it clearer that you use a floating point value.

Reply:
In that tests, FORM does not converge to the probability, that's why the value found by analytical is not compatible with the limitStateTolerance.

Formating typo has been fixed.